libr/core/cmd_info.inc.c

```diff
@@ -1136,6 +1136,12 @@ static void cmd_ic(RCore *core, const char *input, PJ *pj, bool is_array, bool v
 }
 
 static void cmd_iz(RCore *core, PJ *pj, int mode, int is_array, bool va, const char *input) {
+	RBinInfo *info = r_bin_get_info (core->bin);
+	if (!info && pj) {
```

Comment:
izz works on raw binaries. So in this case this condition doesnt handles this situation. Lets drop it for now